core/frontend/src/main/scala/io/udash/utils/Logger.scala

```diff
@@ -30,7 +30,7 @@ private object ConsoleLogger extends Logger {
 }
 
 /**
-  * Provides `logger` reference to [[io.udash.utils.ConsoleLogger]].
+  * Provides `logger` reference to io.udash.utils.ConsoleLogger.
```

Comment:
`Could not find any member to link for "io.udash.utils.ConsoleLogger"` - it's private object.